app/main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+def file_from_input():
+    name = input("Enter name of the file: ")
+    while True:
+        content = input("Enter new line of content: ")
+        if content == "stop":
+            break
+        else:
+            with open(f"{name}.txt", "a") as f:
```

Comment:
It's better to open the file once and then write to it.
Opening the file can be an expensive operation.